pkg/cvo/sync_worker.go

```diff
@@ -231,6 +235,15 @@ func (w *SyncWorker) StatusCh() <-chan SyncWorkerStatus {
 	return w.report
 }
 
+func (w *SyncWorker) StillInitializingFunc() func() bool {
```

Comment:
The only production code callsite calls the returned func right away, do we really need the indirection? I haven't looked at the tests yet but it seems to me we could simply return the bool here.

Reply:
It makes easier to fix the existing unit tests.